lib/commands/prop/prop-print-android-version-code.ts

```diff
@@ -0,0 +1,26 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import {PrintProjectCommand} from "./prop-print";
+
+export class PrintAndroidVersionCodeCommand extends PrintProjectCommand implements ICommand {
+	constructor($staticConfig: IStaticConfig,
+		$injector: IInjector,
+		protected $options: IOptions,
+		private $logger: ILogger,
+		private $projectConstants: Project.IProjectConstants) {
+		super($staticConfig, $injector, $options);
+	}
+
+	execute(args:string[]): IFuture<void> {
+		return ((): void => {
+			super.execute(["AndroidVersionCode"]).wait();
+			if (this.$project.projectData.Framework === this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova && !this.$options.validValue) {
+				this.$logger.info(`Please note that the actual AndroidVersionCode will be ${this.$project.projectData.AndroidVersionCode}2 due to an implementation decision in Apache Cordova.`);
```

Comment:
maybe `printMarkdown` in order to beautify the message that the user will see. I suggest to place `${this.$project.projectData.AndroidVersionCode}2` in \`\`